app/views/your-psp/_stripe.njk

```diff
@@ -91,57 +92,67 @@
     </div>
   </div>
 
- {% else %}
- {% if  enableStripeOnboardingTaskList %}
+{% else %}
+  {% if enableStripeOnboardingTaskList %}
     <h1 class="govuk-heading-m govuk-!-margin-top-8">Information for Stripe</h1>
+
+    {% if progressIndicator.no_of_tasks_completed < progressIndicator.total_number_of_tasks %}
+      <h2 class="govuk-heading-m govuk-!-margin-top-8">Information incomplete</h2>
```

Comment:
<img width="824" alt="Screenshot 2023-02-28 at 17 05 15" src="https://user-images.githubusercontent.com/5648592/221925634-f23ffecb-34cd-4c37-989f-2e3d66c1da6b.png">

The styling doesn't look quite right here. The h2 should have the following classes `<h2 class="govuk-heading-s govuk-!-margin-bottom-2">` according to the design system guidance.

Are we working from a mocked up design from this page? Is it worth checking with Johnny whether we want all the headers? And if so what styling they should have?